modules/core/src/lib/deck.js

```diff
@@ -396,6 +396,20 @@ export default class Deck {
     return views;
   }
 
+  // Returns default viewport based on initialViewState/viewState.
+  _getDefaultViewport(props) {
```

Comment:
This is duplicating what ViewManager does.
Maybe we can send `views` and `viewState` to ViewManager constructor then use `deck.getViewports()` instead?

Reply:
Add a comment where the two instances are initialized on the importance of the order?